cli/tests/integration/run_tests.rs

```diff
@@ -1051,6 +1051,9 @@ fn lock_deno_json_package_json_deps() {
         "jsr:@denotest/module_graph@1.4": "jsr:@denotest/module_graph@1.4.0",
         "npm:@denotest/esm-basic": "npm:@denotest/esm-basic@1.0.0"
       },
+      "jsr": {
+        "@denotest/module_graph@1.4.0": {}
```

Comment:
We always insert these as part of the fix because in the future we'll have an integrity value here.